pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[project]
+name = "center-randomize"
+version = "0.1.0"
+description = "Add your description here"
+authors = [
+    { name = "Manisha Bhandari", email = "manishabhandari841@gmail.com" }
```

Comment:
_Nit_: Probably not needed, commit has all the necessary information

Reply:
I think these informations are necessary as they serve as the main configuration of the project, but open for discussions.

Reply:
I meant the author details

Reply:
project author should be moest-np.